src/components/header/Header.jsx

```diff
@@ -11,18 +13,16 @@ function Header() {
         <section id="header">
             <nav>
                 <div>
-                    <NavLink to="/">Home</NavLink> {' '}
-                    <NavLink to="/income">Income</NavLink> {' '}
-                    <NavLink to="/expenses">Expenses</NavLink> {' '}
-                    <NavLink to="/chart">Chart</NavLink> {' '}
+                    <NavLink to="/">Home</NavLink>
+                    <NavLink to="/income">Income</NavLink>
+                    <NavLink to="/expenses">Expenses</NavLink>
+                    <NavLink to="/chart">Chart</NavLink>
                     <NavLink to="/login">
-                        <button className="btn btn-primary">log-in</button>
+                        <Button name="log-in"/>
                     </NavLink>
-                    {' '}
-                    <NavLink to="/register" className="navbar-right">
-                        <button className="btn btn-primary">Register</button>
+                    <NavLink to="/register">
+                        <Button name="Register"/>
                     </NavLink>
```

Comment:
This is a specific use case whereby you need to follow @noydb comment above about rendering buttons inside of the NavLink, if you want the button styles to be used rather pass the `useNavigate` hook to your button's onClick function so that it navigates when clicking the button.